src/printer.js

```diff
@@ -1952,6 +1952,14 @@ function shouldGroupLastArg(args) {
     (!penultimateArg || penultimateArg.type !== lastArg.type);
 }
 
+function shouldGroupFirstArg(args) {
+  const firstArg = args[0];
```

Comment:
you need to check that there is at least one argument, otherwise it's going to throw an exception `Cannot read property 'type' of undefined`